login/Main.js

```diff
@@ -1,32 +1,17 @@
-const LoginRequest  = require("./LoginRequest.js");
-const UserResponse = require("./UserResponse.js");
-const UserData = require("./UserData.js");
-const UserModel = require("./Models/UserModel.js");
+const singinService= require("./SingIn.js");
+
 
+const LoginRequest  = require("./LoginRequest.js");
+const express = require("express");
+const app = express();
```

Comment:
Maybe you can export this app variable instead of the express one?

Reply:
i will try